src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseHelpAndSupportSection.tsx

```diff
@@ -72,25 +71,25 @@ const ReleaseHelpAndSupportSection = ({
             <p>No methodologies added.</p>
           )}
         </AccordionSection>
-        {release.type.title === ReleaseType.NationalStatistics && (
+        {release.type === 'NationalStatistics' && (
```

Comment:
I was thinking that we should have a type safe way of doing this, rather than having 'NationalStatistics' as a string, but didn't see an easy way of doing it.

Maybe instead of `releaseTypes` being an object, it could be an array of strings - similar to `ImportStatus.tsx`'s `terminalImportStatuses` - might be a bad idea but just throwing it out there.

I also recognise that changing this basically means you have to redo a load of changes, so maybe not worth it regardless.

Would be nice if `release.type` was the same as the string values of `releaseTypes`, but I imagine a load of migration and cache refreshing is required to make that change...